CODENAMES.md

```diff
@@ -11,5 +11,11 @@ releases are subject to change.
 * Hydrogen (18.x 2022)
 * Iron (20.x 2023)
 * Jod (22.x 2024)
+* Krypton (24.x 2025)
+* Lithium (26.x 2026)
```

Comment:
Alternatives:

- Lanthanum
- Lawrencium
- Lead
- Livermorium
- Lutetium

Reply:
I don't think **Lead** has a good connotation though: 

- https://en.wikipedia.org/wiki/Lead#Modern_era
- https://en.wikipedia.org/wiki/Lead%E2%80%93crime_hypothesis

Personally I'd prefer to stick with **Lithium** as proposed here.

Reply:
Lithium salts are also used in the treatment of psychiatric disorders. Whereas lead seems to cause them...

FWIW, lanthanum is a pretty inoffensive element. It isn't even _especially_ toxic.

Reply:
Lithium is also used as a medication for bipolar disorder. Perhaps not the connotation we are hoping for